content/zh/docs/concepts/configuration/manage-resources-containers.md

```diff
@@ -1078,6 +1078,16 @@ spec:
         example.com/foo: 1
 ```
 
+<!--
+## PID limiting
+
+Process ID (PID) limits allow for the configuration of a kubelet to limit the number of PIDs that a given Pod can consume. See [Pid Limiting](/docs/concepts/policy/pid-limiting/) for information.
+-->
+
+## PID限制
+
+进程ID（PID）限制允许对kubelet进行配置，以限制给定Pod可以消耗的PID数量。 有关信息，请参见[Pid Limiting]（/ docs/concepts/policy/pid-limiting/）。
```

Comment:
I could have copy-edited the text if not for my privilege was revoked by someone.
Anyway, this translation looks good. Just some style tweaks needed.

- leave a space between English and Chinese characters, always
- wrap long lines to ease future revision/reviews
- do not replace '(' with '（' for the markdown link because it simply doesn't work
- replace the link target to the zh version because we have translated all those pages

```suggestion
进程 ID（PID）限制允许对 kubelet 进行配置，以限制给定 Pod 可以消耗的 PID 数量。
有关信息，请参见 [PID 限制](/zh/docs/concepts/policy/pid-limiting/)。
```

Reply:
Done
I will keep in mind Anyway, thanks!